README.md

```diff
@@ -49,6 +46,7 @@ and running tests have been implemented.
 | Protectli    | VP4670               | Dasharo                  |  `protectli-vp4670`                    |
 | Raptor-CS    | TalosII              | Dasharo                  |  `raptor-cs_talos2`                    |
 | Raspberry Pi | RaspberryPi 3B       | Yocto                    |  `rpi-3b`                              |
+| Emulation    | QEMU x86             | Dasharo (OVMF)           |  `qemu`                                |
```

Comment:
@macpijan, for some reason, only QEMU has the framework payload naming scheme (`Dasharo (framework+payload)`). I would appreciate the consistency of this table with the [DCS Roadmap](https://github.com/Dasharo/presentations/blob/main/dasharo_roadmap.md#dasharo-emulation-roadmap)
as well as naming, which we introduce in [Dasharo Universe](https://docs.dasharo.com/variants/overview/).

Reply:
2346889fbe5b87eb0a4d184cef4b0dc3fd6a035e

Reply:
@pietrushnic I just wanted to indicate the difference, that no coreboot is used here, and OvmfPkg (not UefiPayloadPkg) from edk2 is used here. How do you suggest to indicate that?

Reply:
@macpijan I'm fine with Dasharo (UEFI). The fact the release is for QEMU Q35 means it has to be OVMF because there is no other way to support EDK II (also incorrectly known as UEFI).

Please check the rules: https://docs.dasharo.com/dev-proc/smbios-rules/#bios-version-field

My comment is not only about Dasharo but also about the fact of how that table is designed. IMHO, it is incorrect and consistent with how we did it in Dasharo Universe. I don't know if we can resolve everything now, but the naming scheme used on Dasharo Universe suggests QEMU should be at the same level as Protectli/Novacustom, and in the platform, you should have Q35 because that machine (in QEMU terminology platform) is emulated and validated.

Reply:
Fixed that line in 38a35a1f4d46